src/components/Note.vue

```diff
@@ -1,6 +1,6 @@
 <template>
-	<NoteRich v-if="rich" :note-id="noteId" />
-	<NotePlain v-else :note-id="noteId" />
+	<NoteRich v-if="ready && rich" :note-id="noteId" />
```

Comment:
I think `v-if="rich"` is enough here because rich already includes condition check of ready (check line 27 and 30).

Reply:
Thanks, pushed a bit of a change there to make the state more clear through computed properties